### PR TITLE
WT-13688 Add total ranges summary to backup test.

### PR DESCRIPTION
**Pull request by sueloverso**
Add a summary line to backup test.

### CODE REVIEW

**Comment by sueloverso** [comment]
This adds a summary statement that makes it easier to measure the impact of changes.

File: examples/c/ex_backup_block.c [hunk 1]
```diff
@@ -310,7 +312,7 @@ take_incr_backup(WT_SESSION *session, int i)
     WT_CURSOR *backup_cur, *incr_cur;
     uint64_t offset, size, type;
     size_t alloc, count, rdsize, tmp_sz;
-    int j, ret, rfd, wfd;
+    int j, range, ret, rfd, wfd;
```
Comment (pmacko86) [inline examples/c/ex_backup_block.c:315]: Nitpick: Maybe use a better name that aligns with the purpose of counting the number of ranges, e.g., `nranges` as in the incremental backup example below? But this is fine too.

**Review by pmacko86 — APPROVED** [review]
LGTM. A small nitpick comment that you are free to ignore. Thanks!

File: examples/c/ex_backup_block.c (hunk 1 above)
Comment (sueloverso) [reply examples/c/ex_backup_block.c:315]: Fixed.

**Pull request merged**